src/System.CommandLine/Option.cs

```diff
@@ -8,22 +8,43 @@
 
 namespace System.CommandLine
 {
+    /// <summary>
+    /// A symbol defining a named parameter and a value for that parameter. 
+    /// </summary>
+    /// <seealso cref="System.CommandLine.IdentifierSymbol" />
+    /// <seealso cref="System.CommandLine.IOption" />
     public class Option :
         IdentifierSymbol,
         IOption
     {
         private string? _implicitName;
         private protected readonly HashSet<string> _unprefixedAliases = new();
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Option"/> class.
+        /// </summary>
+        /// <param name="name">The name of the option, which can be used to specify it on the command line.</param>
+        /// <param name="description">The description of the option shown in help.</param>
+        /// <param name="argumentType">The type that the option's argument(s) can be parsed to.</param>
+        /// <param name="getDefaultValue">A delegate used to get a default value for the option when it is not specified on the command line.</param>
+        /// <param name="arity">The arity of the option.</param>
         public Option(
-            string alias,
+            string name,
             string? description = null,
             Type? argumentType = null,
             Func<object?>? getDefaultValue = null,
             IArgumentArity? arity = null)
-            : this(new[] { alias }, description, argumentType, getDefaultValue, arity)
+            : this(new[] { name }, description, argumentType, getDefaultValue, arity)
         { }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Option"/> class.
+        /// </summary>
+        /// <param name="aliases">The set of strings that can be used on the command line to specify the option.</param>
+        /// <param name="description">The description of the option shown in help.</param>
+        /// <param name="argumentType">The type that the option's argument(s) can be parsed to.</param>
```

Comment:
Can an option have multiple arguments? Or is this intended to refer to the parameters that are parsed in the argument's value?

Reply:
An option can't have multiple instances of `Argument` as children, though it can have multiple tokens parsed toward its one `Argument`.

Reply:
Maybe `argument(s)` -> `argument tokens`?